data/loginUsers.js

```diff
@@ -0,0 +1,78 @@
+const mongoCollections = require("../config/mongoCollections");
+let { ObjectId } = require("mongodb");
+const users = mongoCollections.users;
+const bcrypt = require("bcryptjs");
+const login_validations = require("../validations/login_validations");
+
+let count = 10;
+
+async function checkInput(username, password) {
+  if (!username || !password)
+    throw { code: 400, error: `Username or Password cannot be empty` };
+  if (typeof username != "string")
+    throw { code: 400, error: `Username needs to be a string` };
+  let myUsername = username.split(" ");
+  if (myUsername.length > 1)
+    throw { code: 400, error: `Username cannot have spaces` };
+  if (/[^\w\s]/.test(username))
+    throw { code: 400, error: `Username can only be alphanumeric characters` };
+  if (username.length < 4)
+    throw { code: 400, error: `Username must be at least 4 characters long` };
+
+  if (typeof password != "string")
+    throw { code: 400, error: `Password needs to be a string` };
+  let myPassword = password.split(" ");
+  if (myPassword.length > 1)
+    throw { code: 400, error: `Password cannot have spaces` };
+  if (password.length < 6)
+    throw { code: 400, error: `Password must be at least 6 characters long` };
+}
```

Comment:
This should be in your validation file which you're already importing, so you should just be calling this function from 'login_validations' wherever you're using it in this file

Reply:
completed